google/cloud/storage/blob.py

```diff
@@ -405,7 +405,7 @@ def from_string(cls, uri, client=None):
             >>> from google.cloud import storage
             >>> from google.cloud.storage.blob import Blob
             >>> client = storage.Client()
-            >>> blob = Blob.from_string("gs://bucket/object")
+            >>> blob = Blob.from_string("gs://bucket/object", client=client)
```

Comment:
Thanks for the change! Could we also update both parameter descriptions with a note to strongly encourage specifying a client?

Reply:
745e531